### ISSUE TITLE
Slide Deck Themeing

### ISSUE THREAD

**Issue by wbuchanan** [wbuchanan/sdpConvening2017#28]
I was wondering if maybe we wanted to add another layer of theme to the slide deck?  @daphnejenkins created some for her slides and I was thinking we might organize our material in a way that cycles through a typical project.  For example the first section for each of us would have something like a title slide with `git init` then as we move through different sections we would add a `git add` title slide, have the closing be a `git commit` and then a `git push` to the next presenter.  The next presenter could then start with either a `git init` for a completely different topic or a `git pull` slide if it is building directly on the other person's work.  

Basically I'm just trying to think of other ways that we can build simple reinforcement mechanisms into the flow of the presentation and illustrate the concepts of the different commands in ways that might help make them a bit less abstract.

**Comment by wbuchanan** [comment]
The commit above contains an illustration of the idea I mentioned above.  I branched off of `gh-pages` so you'd be able to pull the branch locally and see what it would look like.  This is also a way that @jsonbecker and I can purposefully create a merge conflict to give people an idea of what that would look like and how to handle it when it does come up.  [Here is the link directly to the new feature branch](https://github.com/wbuchanan/sdpConvening2017/tree/slideIdeas).

**Comment by jsonbecker** [comment]
Are we directly editing the html for this presentation? It would be substantially easier to use R's support for converting RMarkdown to a Reveal presentation...

I think I like this idea, though I think the `vi`/`vim` will be lost on most people.

**Comment by wbuchanan** [comment]
@jsonbecker Sorry thought I got back to this before.  

I had to make some changes to the underlying JS library in order to make the slide deck conform to the template (at least as much as possible).  You're absolutely more than welcome to use R and the `revealjs` package to do things.  I don't mind handling the merges manually or figuring out an easy way to create links to individual slide decks that would sit in the same branch.  

I can get rid of the `vi/vim` stuff....at the very least it is better than having `touch someFile.md` show up all over the place.  I'll see if Roxy has things on the computer she has with her so I can add in some stuff that people can try out with their forks of the repository as well.

**Comment by wbuchanan** [comment]
@jsonbecker 
I can show you the current status of things later today.  To see if it makes sense to you and/or figure out ways that might make it a bit more useful for end users of the slides.

@nathant23 & @daphnejenkins 
Do you want to get together with @jsonbecker and I at around 2:30 PM?  Jason and I are probably going to try connecting a bit earlier to coordinate things a bit more, but figured we could all connect before things kick off a bit more easily than connecting between sessions/dinner.

**Comment by nathant23** [comment]
Yes. Just text me if you want to meet earlier. I'm​ at the hotel now.

**Comment by wbuchanan** [comment]
I added the theme break slides between our respective sections.
